### PR TITLE
more logfetch features (WIP)

### PR DESCRIPTION
**Pull request by ssalinas**

### CODE REVIEW

**Comment by ssalinas** [comment]
@tpetr lots of changes in here, but these should be good to go now if you want to take a look

File: scripts/logfetch/grep.py [hunk 1]
```diff
@@ -2,18 +2,21 @@
 import sys
 from termcolor import colored
 
-GREP_COMMAND_FORMAT = 'xargs -n {0} {1} < {2}'
+GREP_COMMAND_FORMAT = 'while read file; do {0} "$file"; done < {1}'
```
Comment (tpetr) [inline scripts/logfetch/grep.py]: why the change here?

Comment (ssalinas) [inline scripts/logfetch/grep.py]: More complicated custom grep commands weren't playing nicely with xargs. It was more flexible to loop through the file so any grep command that is used will output the way you'd expect it to.
